build/common/installer/scripts/tomlparser.rb

```diff
@@ -59,6 +61,11 @@ def populateSettingValuesFromConfigMap(parsedConfig)
         @collectStdoutLogs = parsedConfig[:log_collection_settings][:stdout][:enabled]
         puts "config::Using config map setting for stdout log collection"
         stdoutNamespaces = parsedConfig[:log_collection_settings][:stdout][:exclude_namespaces]
+        
+        stdoutSystemPods = Array.new
+        if !parsedConfig[:log_collection_settings][:stdout][:collect_system_pod_logs].nil?
+          stdoutSystemPods = parsedConfig[:log_collection_settings][:stdout][:collect_system_pod_logs]
```

Comment:
what would be the behavior if the customer specify the collect_system_pods with not expected format  ? For example, customer specify  ["coredns","metricsserver"] . we would need to validate and log the warn and ignore the invalid entries.